src/CMakeLists.txt

```diff
@@ -147,6 +152,7 @@ SET(LMMS_REQUIRED_LIBS
 	${SAMPLERATE_LIBRARIES}
 	${SNDFILE_LIBRARIES}
 	${EXTRA_LIBRARIES}
+	${KDE_FIX_LIBRARIES}
```

Comment:
The PR can't be merged since rpmalloc. Reabse and just stick rpmalloc to the end.